### PR TITLE
Create release note for new Registration/login rollout

### PR DESCRIPTION
**Pull request by kelly-mcgonnigle**
Closes #

## Summary

Release note for the Auth0 universal login switch, happening next week

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [x ] After date: 6/11 -- confirm with Kelly O'Donnell first

--------------------------------------------------

### CODE REVIEW

File: source/releasenotes/2024-06-12 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Auth0 Universal Login"
+published_date: "2024-06-12"
+categories: [general, security]
+---
+
+Pantheon is enhancing the new user account creation and management process to offer a more stable, modern, and secure first experience. We are moving the registration and login experience to utilize Auth0's Universal Login feature.
```
Comment (stevector) [inline source/releasenotes/2024-06-12:7]: @kelly-mcgonnigle can you add more detail here. Maybe before/after screenshots or links. I don't know, and I don't think most customers will know what "Auth0's Universal Login feature" means

**Comment by stevector** [comment]
I'll bring over the images shared in Slack

**Review by kelly-mcgonnigle — COMMENTED** [review]
looks good to me!

**Review by stevector — APPROVED** [review]

**Review by ABirHAsan123-beep — COMMENTED** [review]
d4f078908b5a2569da7954210421ef64c14936c2

**Pull request merged**